platform/android/java/editor/src/horizonos/java/org/godotengine/editor/GodotXRGame.kt

```diff
@@ -40,7 +40,7 @@ open class GodotXRGame: GodotGame() {
 
 	override fun overrideOrientationRequest() = true
 
-	override fun updateCommandLineParams(args: List<String>) {
+	override fun updateCommandLineParams(args: Array<String>) {
```

Comment:
Why is the method parameter changed to an `Array` instead of a `List`?

Reply:
A list would be mutable (passed by reference), one subtle bug would be if you mutate args inside this function, then use it again outside the function, since there's only one list.

Arrays are immutable so we are forced to create a new array if we want to add to it.